qucs-core/src/components/verilog/analogfunction.xml

```diff
@@ -1090,10 +1108,10 @@ double $(module)_d_$(function) (<admst:join select="variable[input='yes']" separ
 #endif /* __$(module)_ANALOGFUNCTION_H__ */
 <admst:text format="\n"/>
 </admst:open>
-<admst:message format="$(module).analogfunction.h created\n"/>
+<admst:message format="$(filename).$(module).analogfunction.h created\n"/>
 
-<admst:open file="$(module).analogfunction.cpp">/*
- * $(module).analogfunction.cpp - analog function implementations
+<admst:open file="$(filename).$(module).analogfunction.cpp">/*
```

Comment:
The message after the file is created needs to be updated:
https://github.com/Qucs/qucs/pull/294/files#diff-da08accacf866067d2f72d9c2d6c60ffL1118